RELEASING.md

```diff
@@ -89,3 +89,139 @@ and the CI will be green.
+├── edgestack.me
+├── kat
+├── libraries.md
+├── README.md
+└── yaml -> ../submodules/latest/docs/yaml/
+```
+- Now that we have all release branches under `/submodules/`, we further link their `/docs/` directories under `/docs-structure/docs/`.
+```
+docs-structure/docs/
+├── 1.1 -> ../../submodules/1.1/docs
+├── 1.2 -> ../../submodules/1.2/docs
+├── 1.3 -> ../../submodules/1.3/docs
+└── latest -> ../../submodules/latest/docs
+```
+
+This is how our docs are laid out.
+
+In ambassador.git, the CI is configured to update the submodules in getambassador.io.git on every documentation update to the release branches, which deploys to the website.
+
+#### How to host a new release branch.
+
+After a new major/minor release is cut, this is how to host it on the website.
+For example, let's suppose version 1.4 of ambassador needs to be hosted.
+
+##### In ambassador.git,
+
+- In the branch `release/v1.4`,
+  - In `docs/js/doc-page.js`, update Sidebar's prefix to `/docs/latest`
+  ```js
+  <Sidebar location={location} prefix="/docs/latest" items={docLinks} />
+  ```
+  - Update `.travis.yml` to push on every doc change to `release/v1.4` branch.
+  ```yaml
+  deploy:
+  # Add this section
+  - provider: script
+    script: ./.ci/publish-website
+    skip_cleanup: true
+    on:
+      branch: release/v1.4
+  - provider: script
+    script: ./.ci/publish-website
+    skip_cleanup: true
+    on:
+      branch: release/v1.3
+  - provider: script
+    script: ./.ci/publish-website
+    skip_cleanup: true
+    on:
+      branch: release/v1.2
+  ```
+
+##### In getambassador.io.git,
+- Add the submodule pointing to `release/v1.4` branch to the `submodules/1.4/` directory
+```
+git submodule add --name ambassador-1.4 --branch release/v1.4 https://github.com/datawire/ambassador.git submodules/1.4/
+```
+- Update `ambassador-latest` submodule to point to `release/v1.4` branch.
+```
+$ cat .gitmodules
+[submodule "ambassador-latest"]
+        path = submodules/latest
+        url = https://github.com/datawire/ambassador.git
+        branch = release/v1.4
+...
+...
+...
+```
+- Now link only the docs in this branch under `/docs-structure/docs/` directory.
+```
+cd docs-structure/docs/
+ln -s ../../submodules/1.4/docs 1.4
+```
+- Add 1.4 dropdown link in `src/components/Header/Header.js` file.
+```js
+              <li>
+                <div className={classnames(styles.Dropdown, !isDocLink((location || {}).pathname) && styles.hidden)}>
+                  <button className={classnames(styles.DropdownButton, styles.DocsDropdownColor)}>{ docsVersion((location || {}).pathname) } ▾</button>
+                  <div className={styles.DropdownContent}>
+                    <Link to="/docs/latest/">Latest</Link>
+                    <Link to="/docs/1.4/">v1.4</Link>
+                    <Link to="/docs/1.3/">v1.3</Link>
+                    <Link to="/docs/1.2/">v1.2</Link>
+                    <Link to="/docs/1.2/">v1.1</Link>
```

Comment:
Shouldn't this be docs/v1.1? We can probably just keep 1.3, 1.4, and latest here as an example.

Also, since the version numbers are shown without the "v" prefix everywhere else (in the URLs, in links, and besides DOCS once selected), perhaps we should drop it and stick with just 1.4, 1.3, etc..

Reply:
Or, alternatively, include the "v" prefix consistently throughout.

Reply:
Fixed